### ISSUE TITLE
K_PrintScreen doesn't trigger TInputPressRelease event on Debian Linux + XFCE.

### ISSUE THREAD

**Issue by eugeneloza** [castle-engine/castle-engine#37]
Maybe, that's because I have "scrot" assigned to K_PrintScreen in XFCE key bindings? Needs checking.
I've seen this behaviour only in windowed mode. Didn't check it fullscreen.

If it's true then some comment near the definition might come in handy.

**Comment by michaliskambi** [comment]
In X-Windows (your typical GUI environment on desktop Unix), if the window manager catches some key, then it will not reach the final window. This is not specific to PrintScreen, you may as well assign "scrot" to e.g. F5, and then F5 will no reach our window.

There's no way to avoid it on X-Windows...

(

Well, except doing a dirty trick called "override_redirect", which we right now use in the "Xlib backend" in case of fullscreen. But "override_redirect" prevents window manager from handling *any* key combination, including e.g. crucial "Alt + Tab", so it's more of a problem, not a feature:), and in general is not used by any modern application.

Our default backend on Unix, "GTK backend", does not use "override_redirect", regardless of the fullscreen value.

)

Bottom line: this is impossible to fix... All other programs also work like this on X-Windows, you will not catch PrintScreen inside.

**Comment by eugeneloza** [comment]
Thanks a lot! Something like I've suspected. :)
